### ISSUE TITLE
Why is the character's identity poorly maintained?

### ISSUE THREAD

**Issue by zzk88862** [Vchitect/VEnhancer#24]
this is my execution command：
python enhance_a_video.py --version v2 \
--up_scale 2 --target_fps 24 --noise_aug 100 \
--solver_mode 'fast' --steps 15 \
--input_path prompts_v3 \
--prompt_path prompts_v3/text_prompts_v3.txt \
--save_dir 'results/'

input video：

https://github.com/user-attachments/assets/f390379b-a28a-4c5b-98f7-214efd244d1c

output video：

https://github.com/user-attachments/assets/05181ea0-11b8-4526-87fe-56459207fc46
you can see that the identity of woman' face  has changed significantly! can you tell me how to solve this problem? thks

**Comment by hejingwenhejingwen** [comment]
https://github.com/user-attachments/assets/3648945e-adf2-48c7-8ec6-6b69b33d8edb

noise_aug=50; up_scale=4; 
And crop the black borders.

**Comment by zzk88862** [comment]
thanks for your reply

Are you using the v2 model, and is the prompt 'A person strides forward'? 
And Is cropping the black borders to make the video square?

tks

**Comment by hejingwenhejingwen** [comment]
yes, v2 model with prompt "a woman in black is walking towards camera", and crop the black borders before venhancer.
